azure/resource-containers/base-az-resource-container/classes/base-az-resource-container-definition.ps1

```diff
@@ -0,0 +1,8 @@
+Class BaseAzResourceContainerDefinition : PfDefinition{
```

Comment:
Added base resource container classes and module now.  Helps consolidate and refactor code across MG, RG, and Sub